maintainers/maintainer-list.nix

```diff
@@ -14299,6 +14299,12 @@
     githubId = 9636071;
     name = "Myrl Hex";
   };
+  mzabani = {
+    email = "mzabani@gmail.com";
+    github = "mzabani";
+    githubId = 4662691;
+    name = "Marcelo Zabani";
+  };
```

Comment:
please keep this as the first commit, separately:
`maintainers: add mzabani`

Reply:
Thanks for you review! I had to rebase and force-push to change commit history as you suggest in both your comments. Hopefully that's ok.

Reply:
Yes, rebasing is the normal flow used in nixpkgs PRs.